internal/privatestate/data.go

```diff
@@ -0,0 +1,163 @@
+package privatestate
```

Comment:
Let's add a `doc.go` file with a brief Go documentation comment for the package 👍